quality.gradle

```diff
@@ -78,11 +78,13 @@ task pmd(type: Pmd, group: 'verification') {
     }
 }
 
-android {
-    lintOptions {
-        abortOnError true
-        xmlReport true
-        htmlReport true
-        lintConfig file("${project.rootDir}/AndroidCodeQualityConfig/quality/lint/lint.xml")
+afterEvaluate {
```

Comment:
Because If you want to config android lint in multi-project module, you won't get your lint running if you don't wrap android closure in afterEvaluate.Gradle will crash with error `Could not find method android()`.
If we keep these two ways both, I think we can remove `afterEvalate` from `quality.gradle`.